templates/credentials/schedule.html

```diff
@@ -24,31 +21,34 @@ <h5 class="p-notification__title">Error</h5>
   </div>
   {% endif %}
   <div class="row">
+    <div class="col-6">
+      <h2><strong>Schedule your exam</strong></h2>
```

Comment:
This should be `h1` to follow accessibility standards. You can add `class="p-heading--2"` to style it as an `h2`.